src/app/shared/utils/consts.ts

```diff
@@ -53,7 +53,7 @@ export const Consts = {
         "gcp-file" : 'google.svg',
         "yig": "yig.png",
         'alibaba-oss': 'alibaba.svg',
-        
+
```

Comment:
The logo is missing. Please upload the appropriate Sony logo in svg format and push another commit.

Reply:
Will update as soon as we receive the logo.

Reply:
are we taking the logo upload changes with this PR or next PR? If next PR, please open issue to track

Reply:
@kumarashit We are waiting for the logo to be uploaded in this PR. 
If we do not upload the logo then the migration list page will have a missing icon and will break.

Reply:
@anvithks  , @kumarashit  ,  Araki From Sony team will be providing logo/svg file . We are still waiting.

Meanwhile we can open / raise an issue

Reply:
Thank you for this update. Since the logo is required in the migrations list page we will wait for the logo.
There are few PRs to be merged and that will take a few days. We will wait until then. If the logo is not available until then we can merge this PR and raise an issue.